web_development_101/the_backend/ruby_project.md

```diff
@@ -28,7 +28,6 @@ Submit a link below to this [file](https://github.com/TheOdinProject/ruby_course
 * ["Official" Test-First Ruby solutions](https://github.com/ultrasaurus/test-first-teaching/tree/master/learn_ruby) (look for the "solutions" folder inside each exercise)
 
 * Add your solution below this line!
-* [Austin Fisher's solution](https://github.com/austin2016/learn_ruby)
```

Comment:
Hmm, it looks like you're deleting another student's solution. Could you add this back in?